### ISSUE TITLE
Videos being uploaded as "private (locked)"

### ISSUE THREAD

**Issue by Mearman** [ryanfox1985/OctoPrint-Youtube-Timelapse#26]
Possible solution: (https://stackoverflow.com/questions/64079139/using-youtube-data-api-makes-my-videos-private-on-upload/64080239#64080239)

https://discord.com/channels/704958479194128507/704958479709765736/864862967882711052

**Comment by nerddotdad** [comment]
Yup I'm also experiencing this issue. All videos uploaded from my pi are forcibly set to private and I get an email from youtube.

**Comment by msquibb** [comment]
Yup, same thing happening here. Thought it was something that I had setup incorrectly, but sounds like it's not.

**Comment by liquidthex** [comment]
Also experiencing this. Is it possible to incorporate an api-less youtube uploader like this? https://pypi.org/project/youtube-uploader-selenium/

**Comment by AJErazzor** [comment]
I've just set this up and getting it as well

**Comment by cmeans** [comment]
Ditto

**Comment by j-snijder** [comment]
Dito, until a few days ago, now its no longer able to upload.

Dunno what I changed but after lots of marked private, and complaints to youtube, it suddenly did upload and saved to public and was unlocked.
This worked for about a few days and then suddenly (happened a few days ago) its no longer possible to upload.

No idea whats happening but felt obliged to report this here :)

**Comment by cmeans** [comment]
@j-snijder I've found that the upload authentication seems to expire frequently (I've not dug around to see if there's a fix), but simply Deleting the auth and redoing it (which only takes a few minutes), takes care of the problem...for a few weeks.

For whatever reason, my video uploads stopped being flagged as Private and are correctly published as Public now...possibly related to volume :)
